pkg/config/proxy.go

```diff
@@ -4,7 +4,7 @@ package config
 type ProxyConfig struct {
 	StorageType    string `validate:"required" envconfig:"ATHENS_STORAGE_TYPE"`
 	GlobalEndpoint string `envconfig:"ATHENS_GLOBAL_ENDPOINT"` // This feature is not yet implemented
-	Port           string `validate:"required" envconfig:"PORT"`
+	Port           string `validate:"required" envconfig:"ATHENS_PORT" default:":3000"`
```

Comment:
This is really nice. 
I have a few changes to request:
If you have a default value, then the validation should not be required as far as I know. Can you correct me if I am wrong?
Can you add also add a test for this as this is in a critical code section?

Reply:
Makes sense. When I first saw the validate, I thought it might be a way to hook in some kind of field format validation. Doesn't seem to be the case so it can be removed. Will add the test.